main.go

```diff
@@ -148,21 +184,26 @@ func main() {
 			stdoutLogger.Printf("%s = %s", name, value)
 			// Skip if label doesn't match labelWhiteList
 			if !labelWhiteList.Match([]byte(name)) {
-				stderrLogger.Printf("%s does not match the whitelist (%s) and will not be published.", name, whiteListArg)
+				stderrLogger.Printf("%s does not match the whitelist and will not be published.", name)
```

Comment:
We should retain the old behavior of printing the whitelist regular expression to help with debugging. We could still stringify `labelWhitelist`) (https://golang.org/pkg/regexp/#Regexp.String).